tests/integration/test_hyper_transformer.py

```diff
@@ -383,7 +354,7 @@ def test_subset_of_columns():
     """
     data = get_input_data_without_nan()
 
-    ht = HyperTransformer()
+    ht = HyperTransformer(transform_nulls=False)
```

Comment:
Done. This behavior is a little bit misleading though. For example, say the column `b` is `['abc', np.nan]`. If `null_column = True`, then reverse of `b.value` will be `['abc_transformed', 'filled_value']`, but if `null_column = False`, the reverse is `['abc_transformed', np.nan]`.

Basically, when only `column_name.value` is passed, the reverse transform will reverse the null transform if `null_column = False`, and won't otherwise. IMO, this behavior is a little unintuitive, maybe we should just reverse the null transform regardless of whether `b.is_null` is passed.